bait-files/src/main/webapp/success.jsp

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Form Successfully Submitted</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../success.css">
+  </head>
```

Comment:
Would be nice to have the nav bar on this page for visual consistency.

Reply:
I was thinking since it's sort of an inbetween page that shows a message and it navigates back home I left the nav bar out?

Reply:
Does it redirect automatically after awhile? If not a user could just see the checkmark, not read any of the following text and expect to be able to go back to the user history page using the nav bar. It’s nice to have the option and it’s an easy way to know you’re on the same website. Again up to you.

Reply:
yea that's a good point! i'll add a timed redirect